sql/step_91_oh_update_table_laboratory.sql

```diff
@@ -0,0 +1,2 @@
+UNLOCK TABLES;
+ALTER TABLE LABORATORY ADD COLUMN LAB_STATUS VARCHAR(7) NULL DEFAULT NULL;
```

Comment:
Shouldn't there be an update to the demo data to go along with the adding of this column?